### PR TITLE
Test new r2s script

### PR DESCRIPTION
**Pull request by zxkjack123**
Fixed some errors in #45 .
- cell_fracs argument
- code to delete tag

### CODE REVIEW

**Comment by zxkjack123** [comment]
Found bug:
- [x] has bug in sub-voxel r2s cell_fracs tag

**Review by moatazharb — COMMENTED** [review]
Just a comment on few lines that haven't changed in this PR;
In step 2, line 174 under "if subvoxel:", is there a need to reload the geometry to memory and get cell mats again?. Cannot that be returned from step 1?

**Comment by zxkjack123** [comment]
@moatazharb Yes, the "geom" is required to reload in step2. Becacuse the "cell_mats" is  not tagged onto the mesh, it can't be passed from step1 by the h5m file. And the step1 and step2 don't share the memory, so we have to calculate the cell_mats again in stpe2.

**Comment by gonuke** [comment]
This LGTM - has someone actually tested it?

**Comment by zxkjack123** [comment]
@gonuke I tested the code with r2s example files. But I think we should wait for @moatazharb or @ngranda testing it.

**Review by ngranda — COMMENTED** [review]
This looks good to me too. Tested with @zxkjack123 r2s_example

**Comment by moatazharb** [comment]
@zxkjack123 you're correct. I'm wondering, can cell_mats be save to text file in step 1 and loaded by step2 without the need to reload geom and recalculate cell_mats?

**Comment by zxkjack123** [comment]
@moatazharb The cell_mats here is a dictionary of pyne materials. However, these materials don't have an item "id", so, currently I don't know anyway to tag the material ID on the mesh.

**Comment by moatazharb** [comment]
Would it be better to name the config file something like "config_r2s.ini" instead of just "config.ini" since we have other scripts that use config files, so just to avoid confusion.

File: scripts/r2s.py [hunk 1]
```diff
@@ -119,14 +119,24 @@ def step1():
...
                         meshes_have_mats=False)
+        m = meshtal.tally[tally_num]
+
     geom = config.get('step1', 'geom')
     reverse = config.getboolean('step1', 'reverse')
     num_rays = config.getint('step1', 'num_rays')
     grid = config.getboolean('step1', 'grid')
 
     load(geom)
+
```
Comment (moatazharb) [inline scripts/r2s.py]: Probably add a one line descriptive comment here about the following if statement.

File: scripts/r2s.py [hunk 2]
```diff
@@ -119,14 +119,24 @@ def step1():
                                      flux_tag + '_total',
                                      flux_tag + '_err_total')},
                         meshes_have_mats=False)
+        m = meshtal.tally[tally_num]
+
     geom = config.get('step1', 'geom')
```
Comment (moatazharb) [inline scripts/r2s.py]: move all getting input from config file to top before the above if statement.

**Comment by zxkjack123** [comment]
Made some clean up according to the suggestion from @moatazharb .

**Comment by gonuke** [comment]
Thanks all

**Pull request merged**